extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii

```diff
@@ -17,4 +17,23 @@ tags: {  # <amp-story-panning-media>
   amp_layout: {
     supported_layouts: FILL
   }
+  attrs: {
+    name: "data-x"
+    value_regex: "-?\\d+\\.?\\d*(%)"
+  }
+  attrs: {
+    name: "data-y"
+    value_regex: "-?\\d+\\.?\\d*(%)"
+  }
+  attrs: {
+    name: "data-zoom"
+    value_regex: "\\d+\\.?\\d*"
+  }
+  attrs: {
+    name: "lock-bounds"
```

Comment:
It looks like you intend for this to have no value, a presence/missing attr. As written, you allow *any* value. To have no value, specify:

```
value: ""
```

Reply:
Thank you Greg!